website/docs/script/js-script.mdx

```diff
@@ -110,12 +110,20 @@ understanding.
 
 The first step is to create a new Script project. We use [ckb-script-templates](https://github.com/cryptape/ckb-script-templates) for this purpose. You will need the following dependencies:
 
-- `git`, `make`, `sed`, `bash`, `sha256sum` and others Unix utilities
-- `Rust` with `riscv64` target installed: `rustup target add riscv64imac-unknown-none-elf`
-- `Clang 16+`
-- `cargo-generate`: You can install this via `cargo install cargo-generate`
-
-If you encounter any problems with these dependencies, refer to the [ReadMe](https://github.com/cryptape/ckb-script-templates?tab=readme-ov-file#dependencies) for installation details.
+:::info Tools You Need
```

Comment:
can we extract this part to a standalone mdx to be used? seems all script tutorials need this part